### PR TITLE
Loops

### PR DESCRIPTION
**Pull request by acdvs**
This feels right.

Will break other functions in the docs though! Not backwards compatible!

### CODE REVIEW

**Comment by TodePond** [comment]
lgtm!

I think there's a few good jokes to pull out here!

* In most languages, `export` is only available as a top-level keyword. It would be fun to give it more functionality when used at non-top-level (like you've done here).
* There's some fun to be had with `return` being used more literally, or perhaps just introducing some `goto` madness.

I'll have a think!

File: README.md [hunk 1]
```diff
@@ -243,6 +243,28 @@ add(3, 2)!
 
 By the way, to see DreamBerd in action, check out [this page](https://github.com/TodePond/DreamBerd/blob/main/LICENSE.md).
 
+## Loops
+
+Loops can be complicated. The `return` keyword simply returns you to the beginning.
+
+```js
+const var x = 1!
+function loop() => {
```
Comment (artyfarty) [inline README.md:252]: I think `return!` should just go back one line, `return!!` two lines, `return!!!` three lines etc.

Comment (acdvs) [reply README.md:252]: That could open up a huge can of worms for ways to modify other keywords in the same way. Could be revolutionary.

But I also think there would still need to be a way to create a loop like I have here.

**Comment by TodePond** [comment]
I haven't forgotten this PR, I'm just stuck in a bit of a loop about it

**Comment by TodePond** [comment]
I've tidied it up! And now the rest of the codebase needs to be updated to use 'export' instead of 'return', and any loops removed (if there are any)

**Comment by TodePond** [comment]
Ok I think it's ready!

**Pull request merged**